ProjectTerms/Sdl.Community.ProjectTerms.Plugin/ProjectTermsAction/ProjectTermsFilesControllerAction.cs

```diff
@@ -31,5 +37,44 @@ protected override void Execute()
             var parent = projectTermsView.ParentForm;
             projectTermsView.ShowDialog(parent);
         }
-    }
+		private void _filesController_SelectedFilesChanged(object sender, EventArgs e)
+		{
+			// check if selected project having files available or not 
+			var _selectedFiles = _filesController.SelectedFiles.FirstOrDefault();
```

Comment:
You are only checking the first file; please change this logic so it checks if any file exists, then enable = true